Rubberduck.Core/AutoComplete/AutoCompleteFunctionBlock.cs

```diff
@@ -18,10 +18,11 @@ public override bool Execute(AutoCompleteEventArgs e, AutoCompleteSettings setti
             var result = base.Execute(e, settings);
             if (result)
             {
-                var module = e.CodeModule;
+                var module = e.Module;
                 using (var pane = module.CodePane)
                 {
-                    var original = module.GetLines(e.CurrentSelection);
+                    var selection = pane.Selection;
+                    var original = module.GetLines(selection);
                     var hasAsToken = Regex.IsMatch(original, $"\\)\\s+{Tokens.As}", RegexOptions.IgnoreCase) ||
```

Comment:
`\s` matches non-breaking spaces. These should be `[ ]`

Reply:
Thanks - note that block completion hasn't been wired-up or taken care of in any way here, but I'll keep this in mind.